.appveyor.yml

```diff
@@ -26,10 +26,11 @@ install:
           default {$env:MINICONDA = """C:\Miniconda36-x64"""}
       }
   - set PATH=%MINICONDA%;%MINICONDA%\Scripts;%PATH%
+  - set SKLEARN_SITE_JOBLIB=true  # temp fix for joblib warning in examples
```

Comment:
Have no idea why, but **built-in** `joblib` causes warning which transforms into the error on Appveyor. So, lets use `joblib` installed from PyPI.